xdf.cpp

```diff
@@ -735,22 +735,22 @@ void Xdf::findMinMax()
     //find the smallest timestamp of all streams
     for (auto const &stream : streams)
     {
-        if (stream.info.first_timestamp != -1)
+        if (!std::isnan(stream.info.first_timestamp) && stream.info.first_timestamp != -1)
```

Comment:
To be honest, I have no idea why I put `-1` here back then. Seems NAN check is needed here, but the -1 can be taken away? Advice appreciated.

Reply:
No idea why the `-1` and `0` values. There was a discussion in #8 regarding this topic, but I can't remember the details.

Reply:
@cbrnr Yeah looks like a mistake made back then. I have removed these. Tested with a few files and seems working fine.

Reply:
No idea either, but the lsl timestamps are guaranteed to be positive.